components/dashboard/src/projects/projects.routes.ts

```diff
@@ -11,7 +11,7 @@ export const projectsPathMain = "/projects";
 export const projectsPathMainWithParams = [projectsPathMain, ":projectName", ":resourceOrPrebuild?"].join("/");
 
 export const projectsPathInstallGitHubApp = "/install-github-app";
-export const projectsPathNew = "/new";
+export const projectsPathNew = "/projects/new";
```

Comment:
This route makes so much more sense for creating a new project than just `/new` did - thanks 😄